atuin-client/src/import/resh.rs

```diff
@@ -11,6 +11,7 @@ use time::OffsetDateTime;
 use super::{get_histpath, unix_byte_lines, Importer, Loader};
 use crate::history::History;
 
+#[allow(clippy::module_name_repetitions)]
```

Comment:
If there is a good idea for a new name, this would be an easy fix to rename the symbol...